src/attr/__init__.pyi

```diff
@@ -189,6 +199,7 @@ def attrs(
     order: Optional[bool] = ...,
     auto_detect: bool = ...,
     getstate_setstate: Optional[bool] = ...,
+    on_setattr: Optional[_OnSetAttrArgType] = ...,
```

Comment:
Does this actually work?
```
@attr.s(on_setattr=DISABLE)
```

Reply:
It does! Mostly because I didn't want to add a sharp edge for the sake of purity:

https://github.com/python-attrs/attrs/blob/a241f0f152a99e509f572b85d4dd42f82286f1dd/src/attr/_make.py#L800